### PR TITLE
Parallax Photo Carosel for Event Page

### PR DESCRIPTION
**Pull request by Ananya-Misra**
Fixes: #805
The aos effect is not working, I would be glad if you could tell me the reason behind it.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
Highly Unresponsive

**Comment by Ananya-Misra** [comment]
@smaranjitghose Can you please elaborate and provide me with the screenshots for the same.

**Comment by smaranjitghose** [comment]
- Images are too stretched out. Please make them square in shape. This is a fundamental design flaw
Images_to_streched_out


- The size of the buttons are too large for mobile screens
Button_size



- Remove the border around the buttons

- The text color is absolutely weird w.r.t the pictures. This should have been rectified even before giving a PR

File: assets/css/event.css [hunk 1]
```diff
@@ -0,0 +1,346 @@
+@import url('https://fonts.googleapis.com/css2?family=STIX+Two+Math&display=swap');
```
Comment (smaranjitghose) [inline assets/css/event.css:1]: Use Poppins

File: assets/js/app.js [hunk 2]
```diff
@@ -258,7 +258,7 @@ let upArrow = $(`
 //function for the "Scroll To Top" button to detect the footer
 $(document).ready(function () {
   $(window).scroll(function () {
-    console.log($(window).scrollTop());
+    // console.log($(window).scrollTop());
```
Comment (smaranjitghose) [inline assets/js/app.js:261]: Why is this changed?

File: assets/js/event.js [hunk 3]
```diff
@@ -0,0 +1,165 @@
+$(document).ready(function() {
+AOS.init();
+const eventswiper = document.querySelector(".swiper-wrapper");
+const event = [
+    {
+        title : "Slide One",
+        image : "assets/images/event/image3.jpg",
+        text  : "Lorem ipsum, dolor sit amet consectetur adipisicing elit. Possimus optio odit exercitationem, minima non quisquam quibusdam similique error consequatur modi!"
```
Comment (smaranjitghose) [inline assets/js/event.js:8]: Avoid Lorem Ipsum. Put up text like "Talk on XYZ", "Keynote on ABC", "Group Picture"

Only the title is sufficient, Please have a look at the reference

File: event.html [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+        </div>
+   </div>
+
+    <!--Dynamic Nav Bar-->
+
+    <div class="container root_block" style="width: 100%;">
+        <div class="row heading" style= "height: 80vh;">
+            <div data-aos="zoom-in" class="col-lg-6 d-flex flex-column text-center justify-content-center animate__animated animate__zoomIn animate__delay-5s block_one" style= "height: 100%;">
+                <h2 data-aos="fade-up" data-aos-delay="300" class="event_heading">Events</h2>
+                        <p data-aos="fade-right" data-aos-delay="500" class="pre-heading p-upper font-weight-bolder" class="mb-0 mt-3 text-center animate__animated animate__zoomIn animate__delay-6s">
```
Comment (smaranjitghose) [inline event.html:85]: This part is not required. Just replace it with a heading

File: event.html [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+                    <path id="Vector_95" d="M677.87 382.73C681.018 382.73 683.57 380.178 683.57 377.03C683.57 373.882 681.018 371.33 677.87 371.33C674.722 371.33 672.17 373.882 672.17 377.03C672.17 380.178 674.722 382.73 677.87 382.73Z" fill="#916FC1"/>
+                    <path id="Vector_96" d="M692.12 382.73C695.268 382.73 697.82 380.178 697.82 377.03C697.82 373.882 695.268 371.33 692.12 371.33C688.972 371.33 686.42 373.882 686.42 377.03C686.42 380.178 688.972 382.73 692.12 382.73Z" fill="#916FC1"/>
+                    <path id="Vector_97" d="M982.64 546.03C985.788 546.03 988.34 543.478 988.34 540.33C988.34 537.182 985.788 534.63 982.64 534.63C979.492 534.63 976.94 537.182 976.94 540.33C976.94 543.478 979.492 546.03 982.64 546.03Z" fill="#916FC1"/>
+                    <path id="Vector_98" d="M985.49 539.39C988.638 539.39 991.19 536.838 991.19 533.69C991.19 530.542 988.638 527.99 985.49 527.99C982.342 527.99 979.79 530.542 979.79 533.69C979.79 536.838 982.342 539.39 985.49 539.39Z" fill="#916FC1"/>
```
Comment (smaranjitghose) [inline event.html:214]: Not Required

File: event.html [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+                    <stop offset="1" stop-color="#808080" stop-opacity="0.1"/>
+                    </linearGradient>
+                    </defs>
+                    </svg>
+                
+            </div>
+        </div>
+    </div>
+                       
+             <div class="container-fluid">
```
Comment (smaranjitghose) [inline event.html:252]: Maintain Indentation please

File: event.html [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+<br>
+<br>
+
+                 
+
```
Comment (smaranjitghose) [inline event.html:267]: Use some Linters along with Prettier. Avoid unnecessary spaces

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: assets/js/app.js (hunk 2 above)
Comment (Ananya-Misra) [reply assets/js/app.js:261]: I was checking my js function is getting executed or not so I used console and commented it for removing the confusion and forgot to uncomment it.

**Comment by Ananya-Misra** [comment]
@smaranjitghose I got all your points. Thank you for pointing out the flaws. I have just one question, should I go with the reference slider or make some amends in the current one

**Comment by Ananya-Misra** [comment]
@smaranjitghose Thank you for pointing out the flaws. Should I make amends in the current slider or I have to make a whole new slider?

**Comment by smaranjitghose** [comment]
You can make the changes in the same PR

**Comment by Ananya-Misra** [comment]
@smaranjitghose I have made the required changes. Kindly review my PR and inform me in case of any issues.

File: assets/css/event.css [hunk 8]
```diff
@@ -1,37 +1,33 @@
...
+}
 /* variables */
-    html{
-      --dull-white:#ced4e2;
-      --dull-black:#141516;
-      --purple:#8456c4;
-      --white-color: #ffffff;
-      --greyish:#2b2a2A;
-      --green-color:#62aa54;
+html {
```
Comment (smaranjitghose) [inline assets/css/event.css:8]: Refer [this](https://github.com/smaranjitghose/awesome-portfolio-websites/blob/master/assets/css/experience.css) for defining the variables for the CSS file.

Instead of using the names as --green-color make it something more interpretable. For example: --card-bg-color

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
The changes indeed are a great improvement.

update

- Please shift the text towards the bottom as shown above 
- Resize the cards/images so that they appear to be square in shape
- During the change of images, only the next image should be displayed but for this one, it feels like the toggle buttons disappear with the images and reappear. 

carosel_movement

**Comment by anushbhatia** [comment]
Keep the image resolution to be the same so that with any image size this carousel can work.

**Comment by Ananya-Misra** [comment]
@smaranjitghose I made the required changes but I didn't change the shape of the image cause on changing the shape of the image to square the UI looks like this:
image

**Review by smaranjitghose — APPROVED** [review]
